netman/adapters/switches/cached.py

```diff
@@ -316,6 +316,14 @@ def unset_interface_lacp_force_up(self, interface_id):
         self.real_switch.unset_interface_lacp_force_up(interface_id)
         self.interfaces_cache[interface_id].force_up = None
 
+    def set_interface_recovery_timeout(self, interface_id, recovery_timeout):
```

Comment:
Are you missing the unset ? Should it be cleaned during interface de-configuration ?

Reply:
thank you! @fbouliane , I have updated the cached interfaces

regarding unset, had this concern - so I had verified with nico it cleans up everything along with other options in reclaiming

ideally we could have a separate api to remove specifically this parameter, but not required by anything else at this point